docs/CUSTOM_RULES_101.md

```diff
@@ -553,63 +635,75 @@ $ mvn clean install -f pom_SQ_8_9_LTS.xml
 [INFO] ------------------------------------------------------------------------
 ```
 
-Then, grab the jar file `java-custom-rules-example-1.0.0-SNAPSHOT.jar` from the `target` folder of the project, and move it to the extensions folder of your SonarQube instance, which will be located at `$SONAR_HOME/extensions/plugins`.
+Then, grab the jar file `java-custom-rules-example-1.0.0-SNAPSHOT.jar` from the `target` folder of the project, and move it to the extensions folder of your SonarQube instance, which will be located at `$SONAR_HOME/lib/extensions`.
 
 >
-> :exclamation: **SonarQube Java Plugin compatible version**
+> Exclamation: **SonarQube Java Plugin compatible version**
```

Comment:
It's not the word `exclamation` surrounded by `:` but the emoji :exclamation:, emojis are a small images used by young people to express an idea or an emotion :trollface:
```suggestion
> :exclamation: **SonarQube Java Plugin compatible version**
```